src/Cooked/Skeleton.hs

```diff
@@ -870,8 +870,9 @@ paysPK pkh value =
         (Nothing @(Script.Versioned Script.Script))
     )
 
--- | Pays a script a certain value with a certain datum, using the
+-- | Pays a script a certain value with a certain datum hash, using the
```

Comment:
This is the third time I write a comment about this line of comment. There is something bothering me, which I find unclear, but I can't really put my finger on it. Here is another suggestion which barely satisfies me more.

"Issues a new UTxO at a script address. The UTxO only carries the hash of its provided datum. It is resolved inside the transaction that generates the UTxO."

This is extreme nitpicking anyway.